### PR TITLE
Accessibility | Starters/Leavers/Vacancies Labels

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[Accessibility - Starters/Leavers/Vacancies - Missing Form Labels](https://trello.com/c/C8nFVtY8/204-accessibility-starters-leavers-vacancies-missing-form-labels)

File: src/app/features/workplace/vacancies/vacancies.component.html [hunk 1]
```diff
@@ -17,10 +17,10 @@ <h1 class="govuk-fieldset__heading">Add current staff vacancies</h1>
     <ng-container formArrayName="vacancies">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds govuk-form-group govuk-!-margin-bottom-0">
-          <p>Job Role</p>
+          <label for="jobRole-0" class="govuk-label">Job Role</label>
```
Comment (omarCreativeDev) [inline src/app/features/workplace/vacancies/vacancies.component.html]: maybe its better to have an visually invisible `<label>` tag next to each form element?

Comment (ChazUK) [reply src/app/features/workplace/vacancies/vacancies.component.html]: This has come straight from Louise, I'm not happy with the implementation or the design for accessibility, but she says that if these first labels are in place she will pass it 🤷‍♂

Comment (omarCreativeDev) [reply src/app/features/workplace/vacancies/vacancies.component.html]: okie

**Pull request merged**